applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -100,6 +110,15 @@ def __init__(self, model, custom_settings):
 
         self._bfecc_convection = self.settings["bfecc_convection"].GetBool()
 
+        # this is used to perform one step BFECC for the eulerian LS convection
+        self._eulerian_error_compensation = self.settings["levelset_convection_settings"]["eulerian_error_compensation"].GetBool()
```

Comment:
This variable is not needed if the convection is done with BFECC.

Reply:
As mentioned above we can merge this with "bfecc_convection" in a future PR.